retrofit/src/main/java/retrofit/OkHttpBodyConverterFactory.java

```diff
@@ -28,8 +30,9 @@
     if (RequestBody.class.isAssignableFrom(cls)) {
       return new OkHttpRequestBodyConverter();
     }
-    if (ResponseBody.class.isAssignableFrom(cls)) {
-      return new OkHttpResponseBodyConverter(false);
+    if (cls == ResponseBody.class) {
```

Comment:
Slight behavior change here for the better. If you used `MyResponseBody` we would incorrectly match it as valid and then blow up trying to cast a different subclass.